Src/Support/Google.Apis.Auth/JsonWebToken.cs

```diff
@@ -94,6 +94,13 @@ public class Payload
             [Newtonsoft.Json.JsonPropertyAttribute("jti")]
             public string JwtId { get; set; }
 
+            /// <summary>
+            /// The nonce value specified by the client during the authorization request.
+            /// Must be present if a nonce was specified in the authorization request, otherwise this will not be present.
+            /// </summary>
+            [Newtonsoft.Json.JsonProperty("nonce")]
```

Comment:
Possibly JsonPropertyAttribute for consistency with the others, or fix all of them to just say JsonProperty?

Reply:
Thanks, done.